Contest_Archive_Format.md

```diff
@@ -85,7 +85,7 @@ The following formats may be used:
 
 | ID     | Description                                               | Specification                                           |
 | ------ | --------------------------------------------------------- | ------------------------------------------------------- |
-| kattis | Kattis problem package format                             | <http://www.problemarchive.org/>                        |
+| kattis | Kattis problem package format                             | <https://www.problemarchive.org/>                        |
```

Comment:
Now available at https://www.kattis.com/problem-package-format/ (although that might change?)

Reply:
So what is your requested change then exactly? Change it to that URL for now? Likely you can at least keep it working as a redirect even if it changes, of course.

Reply:
Yeah... that's a fair question :). Sorry that was very unclear of me.

I would suggest changing it to `https://www.kattis.com/problem-package-format/` now and just updating it again if and when it changes. 

That said, if you strongly prefer to leave it as is for now, just dismiss my change request and I will merge anyway.